Workbooks/Windows Virtual Desktop/shared/Template Versioning Footer/Template Versioning Footer.workbook

```diff
@@ -13,7 +13,8 @@
             "type": 1,
             "description": "Internal parameter to centralize the template version",
             "isRequired": true,
-            "value": "0.51.1",
+
+            "value": "0.51.1.210317",
```

Comment:
Something is missing here, this is not the latest version

Reply:
Fixed